akka-docs/src/main/paradox/stream/operators/Sink/cancelled.md

```diff
@@ -13,6 +13,16 @@ Immediately cancel the stream
 
 Immediately cancel the stream
 
+## Example
+
+In this example, we have a source that generates numbers from 1 to 5 but as we have used cancelled we get NotUsed as return type and stream cancels.
```

Comment:
Maybe:

```suggestion
In this example, we have a source that generates numbers from 1 to 5 but as we have used cancelled we get NotUsed as materialized value and stream cancels.
```

It would be nice to have an example that more clearly illustrates how this compares to `Sink.ignore`... but I can't come up with anything nice either.

Reply:
I will think of such example and if something comes up in my mind... then I'll update this